crates/bitcoind_rpc/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "bdk_bitcoind_rpc"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+bdk_chain = { path = "../chain", version = "0.5.0", features = ["serde", "miniscript"] }
+bitcoincore-rpc = { version = "0.17.0" }
+anyhow = { version = "1" }
```

Comment:
Do we need to add `anyhow` ? None of the other crates use it, so would like to not have to add it for this one.

Reply:
Woops, I think that was only meant as a dev dependency! Thanks for pointing this out.